### PR TITLE
feat(field): apply ODS theme

### PR DESCRIPTION
**Pull request by AxelPeter**
- Apply ODS theme to `oui-field`
- Update documentation

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report yarn-build#1835.0 ✘  
* build & test
  * build ✔ 
  * test ✘

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci#1838.0 ✘  
* 
  * deploy_uxci ✘

File: packages/components/field/src/js/field.html [hunk 1]
```diff
@@ -1,33 +1,35 @@
...
+                </span>
             </span>
-        </span>
-    </div>
-    <div class="oui-field__helper"
-        ng-if="!!$ctrl.helpText"
-        ng-bind="$ctrl.helpText">
+        </div>
+        <div class="oui-field__helper"
+            ng-if="!!$ctrl.helpText"
```
Comment (FredericEspiau) [inline packages/components/field/src/js/field.html]: Maybe you could add `data-` to attributes in `div` elements as it's not HTML valid (I know it's like that everywhere in ui-kit right now 😅 )

Comment (AxelPeter) [reply packages/components/field/src/js/field.html]: It has been already been discussed a multiple time, and the answer is `No` ;)

**Pull request merged**